calour/heatmap/heatmap.py

```diff
@@ -265,24 +267,36 @@ def _ax_color_bar(ax, values, width, position=0, colors=None, axis=0, label=True
         the width of the color bar
     position : float, optional
         the position of the color bar (its left bottom corner)
-    colors : list of colors, optional
+    colors : dict or pd.Series, optional
         the colors for each unique value in the ``values`` list.
         if it is ``None``, it will use ``Dark2`` discrete color map
         in a cycling way.
     horizontal : bool, optional
         plot the color bar horizontally or vertically
     label : bool, optional
         whether to label the color bars with text
+    **label_kwargs: dict
+        keyword arguments to pass in for annotating labels
 
     Returns
     -------
     ``matplotlib`` axes
     '''
+
+    default_kwargs = {'color': 'w', 'weight': 'bold', 'fontsize': 7,
+                      'ha': 'center', 'va': 'center'}
+    for k, v in default_kwargs.items():
+        if k not in label_kwargs.keys():
+            label_kwargs[k] = v
```

Comment:
I believe you can update `default_kwargs` with `dict.update` func? to simplify the code

Reply:
That is awesome!  I didn't know that existed